services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/modules/db/repositories/osparc.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import sqlalchemy as sa
+from models_library.projects import ProjectID
+from simcore_postgres_database.models.projects import projects
+from simcore_postgres_database.models.users import users
+
+from ._base import BaseRepository
+
+_logger = logging.getLogger(__name__)
+
+
+class OSparcRepository(BaseRepository):
+    async def get_user_email(self, user_id: int) -> str | None:
+        async with self.db_engine.connect() as conn:
+            result = await conn.execute(
+                sa.select(users.c.email).where(users.c.id == user_id)
+            )
+            data = result.first()
+            output: str | None = data[0] if data else None
+            return output
+
+    async def get_project_name_and_workbench(
```

Comment:
suggestion: separate concerns
for now it has only 2 methods (which are probably also available somewhere else), so it's ok. but calling this osparc.py is conflicting with the osparc-api-client at least among other things.
Since we already discussed, maybe we should move that down a level and have these repos in postgres-database. @pcrespov?

Reply:
Will rename it, in the future we can utilize repos from Postgres-database